tasks/strings/strings.js

```diff
@@ -22,9 +33,23 @@ function reverseString(str) {
  * console.log(centuryFromYear(1601)) // 17
  * hint: please choose and appropriate method from the Math object
  */
-function centuryFromYear(year) {
-
-}
+    function centuryFromYear(year) {
+      if (year >= 1601 && year <= 1700) {
```

Comment:
it can be done in one line using Math.ceil() method: https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Math/ceil